### PR TITLE
Bugfix 1314

### LINKED ISSUE #1314: TypeError: Cannot read property 'map' of undefined
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)

0.19

**What operating system and version?**

All

**What did you do?**

```
Processes: window, platforms: darwin linux win32, versions: 0.18.0 0.19.0
TypeError: Cannot read property 'map' of undefined
    at TorrentListController.resumePausedTorrents (...\build\renderer\controllers\torrent-list-controller.js:187:38)
    at EventEmitter.<anonymous> (...\build\renderer\main.js:391:76)
    at emitThree (events.js:121:20)
    at EventEmitter.emit (events.js:194:7)
```

This is the top error in the telemetry logs since the release of 0.19. Appears to be affecting A LOT of users. @codealchemist - any ideas?

### PR DESCRIPTION
**Pull request by codealchemist**
Fixes #1314

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent-desktop",
   "description": "WebTorrent, the streaming torrent client. For Mac, Windows, and Linux.",
-  "version": "0.19.0",
+  "version": "0.19.1",
```
Comment (DiegoRBaquero) [inline package.json]: Never bump versions in a PR, this is done when releases are about to be done.

Comment (codealchemist) [reply package.json]: Ouch, I'm too used to that.
I'll update it.
Thanks!

**Comment by bnjmnt4n** [comment]
Was just looking at the code and wondering: it is necessary for this line to be called: https://github.com/webtorrent/webtorrent-desktop/pull/1316/files#diff-560dc6e1bfb4ac8e064a21c5f1118156R183?

**Comment by codealchemist** [comment]
Hi @bnjmnt4n!
This is to ensure there are torrents to resume before trying to resume:
`if (!this.state.saved.torrentsToResume || !this.state.saved.torrentsToResume.length) return`

Every time the player is closed, we try to resume paused torrents and that conditional is needed to avoid errors in the case we never added torrents to the resume collection.

**Comment by codealchemist** [comment]
@DiegoRBaquero can you test this fix?

**Pull request merged**